type errors in v2.0.0

The following minimal setup does not build without type errors for me.

index.ts
```
import { getMockReq, getMockRes } from '@jest-mock/express'

getMockReq()
```

tsconfig.json
```
{
    "compilerOptions": {
      "allowSyntheticDefaultImports": true,
      "lib": ["es2021"],
      "module": "es2020",
      "moduleResolution": "node",
      "target": "es2021"
    }
}
```

package.json
```
{
  "name": "foo",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "tsc": "tsc -p ."
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@jest-mock/express": "^1.4.5",
    "@types/express": "^4.17.13",
    "@types/jest": "^28.1.4",
    "jest": "^28.1.2",
    "typescript": "^4.7.4"
  },
  "dependencies": {}
}
```

results in:
```
node_modules/@jest-mock/express/dist/src/request/index.d.ts:7:40 - error TS2307: Cannot find module 'src' or its corresponding type declarations.

7 import type { EventEventEmitter } from 'src';
                                         ~~~~~

node_modules/@jest-mock/express/dist/src/request/request.d.ts:6:117 - error TS2304: Cannot find name 'ParsedQs'.

6 export declare const getMockReq: <T extends Request<import("express-serve-static-core").ParamsDictionary, any, any, ParsedQs, Record<string, any>>>(values?: MockRequest) => T;
                                                                                                                      ~~~~~~~~

node_modules/@jest-mock/express/dist/src/response/index.d.ts:7:40 - error TS2307: Cannot find module 'src' or its corresponding type declarations.

7 import type { EventEventEmitter } from 'src';
                                         ~~~~~


Found 3 errors in 3 files.

Errors  Files
     1  node_modules/@jest-mock/express/dist/src/request/index.d.ts:7
     1  node_modules/@jest-mock/express/dist/src/request/request.d.ts:6
     1  node_modules/@jest-mock/express/dist/src/response/index.d.ts:7
```

Not sure what I'm missing here but the pathes to 'src' really seem off?

The problem does not occur with v1.4.5

--- a/package.json
+++ b/package.json
@@ -42,7 +42,6 @@
   "homepage": "https://github.com/bikk-uk/jest-mock-express#readme",
   "devDependencies": {
     "@snyk/protect": "^1.947.0",
-    "@types/express": "^4.17.13",
     "@types/jest": "^27.5.1",
     "@typescript-eslint/eslint-plugin": "^5.27.1",
     "@typescript-eslint/parser": "^5.27.1",
@@ -54,6 +53,8 @@
     "ts-jest": "^27.1.3",
     "typescript": "^4.7.3"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@types/express": "^4.17.13"
+  },
   "snyk": true
 }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@jest-mock/express",
-  "version": "2.0.0",
+  "version": "2.0.1",
   "description": "A lightweight Jest mock for unit testing Express",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",

--- a/src/request/index.ts
+++ b/src/request/index.ts
@@ -2,7 +2,9 @@
 import type { IncomingMessage } from 'http'
 import type { Readable } from 'stream'
 import type { Request } from 'express-serve-static-core'
-import type { EventEventEmitter } from 'src'
+
+// Local Types
+import type { EventEventEmitter } from '../index'
 
 interface StreamReadable extends EventEventEmitter {
   readable?: Readable['readable']

--- a/src/request/request.ts
+++ b/src/request/request.ts
@@ -1,7 +1,7 @@
-// Libraries
-import { Request } from 'express'
-
 // Types
+import type { Request } from 'express'
+
+// Local Types
 import type { MockRequest } from './index'
 
 /**

--- a/src/response/index.ts
+++ b/src/response/index.ts
@@ -2,7 +2,9 @@
 import type { ServerResponse, OutgoingMessage, IncomingMessage } from 'http'
 import type { Writable } from 'stream'
 import type { Response } from 'express'
-import type { EventEventEmitter } from 'src'
+
+// Local Types
+import type { EventEventEmitter } from '../index'
 
 interface StreamWritable extends EventEventEmitter {
   writable?: Writable['writable']

--- a/src/response/response.ts
+++ b/src/response/response.ts
@@ -1,7 +1,7 @@
-// Libraries
-import { Response, NextFunction } from 'express'
-
 // Types
+import type { Response, NextFunction } from 'express'
+
+// Local Types
 import type { MockResponse } from './index'
 
 /**

--- a/src/test/request/github.test.ts
+++ b/src/test/request/github.test.ts
@@ -7,7 +7,7 @@ import getMockReq from '../../request/request'
 describe('request - GitHub Issues', () => {
   test('issue #6', () => {
     enum AppOS {
-      Android = 1,
+      Android = '1',
     }
 
     const req = getMockReq({

--- a/src/test/request/provided/provided.test.ts
+++ b/src/test/request/provided/provided.test.ts
@@ -24,8 +24,8 @@ describe('request - Provided (accepts arguments and returns expected values)', (
       user: mockUser,
       query: {
         id: '123',
-        limit: 10,
-        page: 2,
+        limit: '10',
+        page: '2',
       },
     })
 
@@ -38,7 +38,7 @@ describe('request - Provided (accepts arguments and returns expected values)', (
     expect(req.query).toBeInstanceOf(Object)
     expect(Object.keys(req.query).length).toBe(3)
     expect(req.query['id']).toBe('123')
-    expect(req.query['limit']).toBe(10)
-    expect(req.query['page']).toBe(2)
+    expect(req.query['limit']).toBe('10')
+    expect(req.query['page']).toBe('2')
   })
 })

